inc/certificate.class.php

```diff
@@ -689,9 +689,6 @@ static function cronCertificate($task = null) {
       }
 
       $message      = [];
-      $items_notice = [];
```

Comment:
seems strange, but i checked, and it's ok

Reply:
Yep; they were not used